app/src/context-engine/tests/integration/test_cli_auth_linear_e2e.py

```
@@ -0,0 +1,120 @@
+
+    tokens: dict[str, Any] = {
+        "access_token": access,
+        "expires_at": 9999999999.0,
+    }
+    refresh = _env("CLI_AUTH_E2E_LINEAR_REFRESH_TOKEN")
+    if refresh:
+        tokens["refresh_token"] = refresh
+
+    save_integration_tokens("linear", tokens)
+
+    proc = _run_potpie("--json", "auth", "status", "--verify", env=isolated_cli_env)
```

Ruff (0.15.15)

[error] Undefined name `_run_potpie`

(F821)